Makefile

```diff
@@ -206,7 +206,7 @@ pytest: biolink/model.py
 	pipenv run python $<
 
 jsonschema_test: json-schema/biolink-model.json
-	jsonschema $<
+#	jsonschema $<
 
```

Comment:
why are we removing a valid test?

Reply:
The test didn't exist in the master branch.  Would be happy to have it _become_ a valid test...

Reply:
Perhaps some folks have "jsonschema" installed natively on their machines?

Reply:
Yes, jsonschema is a dependency that is not well documented. It is external to biolinkml.

Reply:
Can we add a script to install it in the Makefile itself?

Reply:
I'll remove it from the pull request for the time being as long as the Makefile works elsewhere

Reply:
Might be safer to do a check for `jsonchema` executable and then run this test only if it exists.

Reply:
Our GitHub Actions does install jsonschema as part of its workflow. So we will always be running this